Gruntfile.js

```diff
@@ -21,11 +21,12 @@ module.exports = function (grunt) {
         }
     });
 
-    grunt.loadNpmTasks('grunt-contrib-jshint');
-    grunt.loadNpmTasks('grunt-contrib-nodeunit');
-    grunt.loadNpmTasks('grunt-contrib-clean');
+    // grunt.loadNpmTasks('grunt-contrib-jshint');
+    // grunt.loadNpmTasks('grunt-contrib-nodeunit');
```

Comment:
@fabe85 - You are right.. Ideally, we should delete obsolete comments. However, I felt these in the Gruntfile for now, because I thought we might implement clean, hint and test shortly.. (before 7th)..
But maybe its better to remove it now.. can always get it back!
I'll delete them in my next commit...
Thanks.